src/components/FeaturePanel/ImageSection/FeatureImage.tsx

```diff
@@ -7,34 +7,23 @@ import { Feature } from '../../../services/types';
 import { InlineSpinner } from './InlineSpinner';
 import { t } from '../../../services/intl';
 import { nl2br } from '../../utils/nl2br';
+import { Photo } from './Photo';
 
 const Wrapper = styled.div`
   position: relative;
-  background: #ddd url(${({ link }) => link ?? ''}) center center no-repeat;
-  background-size: ${({ portrait }) => (portrait ? 'contain' : 'cover')};
+  background: #ddd
+    linear-gradient(
+      to bottom,
+      rgba(0, 0, 0, 0) 70%,
+      rgba(0, 0, 0, 0.09) 76%,
+      #5b5b5b
+    );
   height: 238px;
   min-height: 238px; /* otherwise it shrinks b/c of flex*/
   ${({ uncertainImage }) =>
     uncertainImage
       ? 'box-shadow: inset 0 0 100px rgba(255,255,255,0.3); filter: contrast(0.6) brightness(1.2);'
       : ''}
-
-  &:before {
```

Comment:
This :before suits as a cover over the image, which helps the poi type to be better readable. You can perhaps add it as additional element and set pointer-events accordingly.

I see you added the gradient to the #ddd background, but it is not the same effect 🙂 

See missing shadow here:
<img src="https://github.com/zbycz/osmapp/assets/385047/500cad3f-b01c-4892-b235-8398b2833e05" width="300">